server/server.go

```diff
@@ -365,6 +369,17 @@ func (s *Server) Kill(connectionID uint64, query bool) {
 	}
 }
 
+// KillAllConnections kills all connections when server is not gracefully shutdown.
+func (s *Server) KillAllConnections() {
```

Comment:
Call cancel function in clientConn.dispatch can make the connection return?

Reply:
Call `ctx.cancel` will  cancel the current execution, `atomic.StoreInt32(&conn.status, connStatusWaitShutdown)` will make the connection close itself later.